packages/pwa-kit-create-app/assets/pwa/default.js

```diff
@@ -6,9 +6,38 @@
  */
 exports.template = ({commerceApi, einsteinApi}) => `module.exports = {
     app: {
-        url: {
-            locale: 'none'
+        "url": {
+            // "site": "none",
+            // you can set how you want your locale to display in the url as one of "path|query_param|none"
+            "locale": "none",
+            // a flag to display default values (either site and locale) to display in the url
+            // if they are set to be in path or query_param. In most cases, you might want to
+            // set it to true.
+            // "showDefaults": true
         },
+        "defaultSite": "RefArch",
+        // "siteAliases": {
+        // you can add alias to your site id
+        //     "RefArch": "us"
+        // },
+        "sites": [
+            {
+                "id": "RefArch",
+                "l10n": {
+                    "supportedCurrencies": ["USD"],
+                    "defaultCurrency": "USD",
+                    "defaultLocale": "en-US",
+                    "supportedLocales": [
+                        {
+                            "id": "en-US",
+                            // you can add alias for your locale
+                            // "alias": "us",
+                            "preferredCurrency": "USD"
+                        }
+                    ]
+                }
+            }
+        ],
```

Comment:
A couple comments here to maybe make things look a little more polished. 
1. We need to be consistent on whether the comment in above or below.
2. If we have comments for some, we should comment them all. (Complex objects like the sites array can probably use one block comment on top.
3. It will be nice let the reader know what the default values are. 
4. It looks like you are still using JSON syntax with the double-quote keys etc, we'll want to address that.
5. It will probably look nicer if you start your sentences with capitals :)

I played around with it a little and here is an example of what some of the changes might look like:


```
    app: {
        // Customize how your 'site' and 'locale' are displayed in the url.
        url: {
            // site: 'none',            // Where the current site identifier is displayed. Valid values include 'path|query_param|none'. Defaults to: 'none'
            locale: 'none',             // Where the current locale identifier is displayed. Valid values include 'path|query_param|none'. Defaults to: 'none'
            // showDefaults: true       // This boolean value dictates whether or not default site or locale values are shown in the url. Defaults to: false
        },
        defaultSite: 'RefArch',
        // Provide alias's for your sites. These will be used in place of your site id when generating paths throughout the application.
        // siteAliases: {
        //     RefArch: 'us'
        // },
        ...
```